daphne_worker/src/lib.rs

```diff
@@ -512,6 +519,70 @@ pub(crate) struct InternalTestAddTask {
     task_expiration: Time,
 }
 
+/// LogWriter helps us write to the worker console.
+///
+/// It provides and io::Write implementation that provides line
```

Comment:
nit: My rust linter wraps  comments at 100 characters.

Reply:
I fixed this and adjust my ide though we may want to consider turning on autoformatting of comments by the rustfmt.